docker/Dockerfile.fedora

```diff
@@ -0,0 +1,23 @@
+FROM fedora:31
+
+ARG user_id=1000
+ENV env_user_id=${user_id}
+
+RUN groupadd -g ${env_user_id} conan && \
+    useradd -r -u ${env_user_id} -g conan conan
+
+USER root
+WORKDIR /home/conan
```

Comment:
This directory needs to exist in order to change to it.

Reply:
I added the -m option to the useradd command